frontend_app_file/src/container/File.jsx

```diff
@@ -166,9 +167,11 @@ export class File extends React.Component {
     }))
   }
 
-  handleContentCreated = (data) => {
-    if (data.content.content_type === 'comment' && data.content.parent_id === this.state.content.content_id) {
-      const sortedNewTimeLine = sortTimelineByDate([...this.state.timeline, { ...data.content, created_raw: data.content.created }])
+  handleContentCommentCreated = (data) => {
+    if (data.content.parent_id === this.state.content.content_id) {
+
+      const sortedNewTimeLine = sortTimelineByDate([...this.state.timeline, { ...data.content, created_raw: data.content.created, timelineType: data.content.content_type }])
```

Comment:
avoid too long lines
```suggestion
      const sortedNewTimeLine = sortTimelineByDate([
        ...this.state.timeline,
        {
          ...data.content,
          created_raw: data.content.created,
          timelineType: data.content.content_type
        }
      ])
```

Reply:
Or

```suggestion
      const sortedNewTimeLine = sortTimelineByDate([...this.state.timeline, {
        ...data.content,
        created_raw: data.content.created,
        timelineType: data.content.content_type
      }])
```

Both are standard.js valid. The choice is yours